dashboards/sql/qualified_benefits.sql

```
@@ -1,12 +1,15 @@
 WITH totals AS (
-    SELECT count(*) as total_count FROM analytics.mart_screener_data WHERE 1=1 [[AND {{partner}}]]
+    SELECT count(*) as total_count FROM analytics.mart_screener_data WHERE 1=1 [[AND {{partner}}]] [[AND {{county}}]]
```

SQLFluff (4.0.4)

[error] Keywords must be upper case.

(CP01)